Mathlib/Analysis/SpecialFunctions/Complex/Arg.lean

```diff
@@ -346,6 +353,20 @@ theorem arg_inv (x : ℂ) : arg x⁻¹ = if arg x = π then π else -arg x := by
   · exact arg_real_mul (conj x) (by simp [hx])
 #align complex.arg_inv Complex.arg_inv
 
+@[simp] lemma abs_arg_inv (x : ℂ) : |x⁻¹.arg| = |x.arg| := by rw [arg_inv]; split_ifs <;> simp [*]
+
+lemma abs_eq_one_iff' : abs x = 1 ↔ ∃ θ ∈ Set.Ioc (-π) π, exp (θ * I) = x := by
```

Comment:
Could you please add a version saying `(fun θ ↦ exp (θ * I)) '' Set.Ioc (-π) π = Metric.sphere 0 1`? BTW, this should be a general fact about `Periodic` functions (image = range).

Reply:
I agree that it should be there in greater generality, but I found the glue painful to write. For now, I've left a TODO.